kolibri/plugins/pdf_viewer/assets/src/views/PdfRendererIndex.vue

```diff
@@ -281,11 +281,9 @@
         // Is either the first page or the saved last page visited
         const firstPageToRender = parseInt(this.getSavedPosition() * this.totalPages);
         return this.getPage(firstPageToRender + 1).then(firstPage => {
-          // page.view is a viewbox array of [x1, y1, x2, y2] coordinates where x1, y1 is the
-          // top left corner and x2, y2 is the bottom right corner of the visible page in PDF
-          // coordinates, subtracting the two gives us the width and height of the visible page
-          this.firstPageHeight = firstPage.view[3] - firstPage.view[1];
-          this.firstPageWidth = firstPage.view[2] - firstPage.view[0];
+          const viewPort = firstPage.getViewport({ scale: 1 });
```

Comment:
`scale: 1` since here we still don't have the final scale value that will be used to adjust to elementWidth.